ParisBNF/abb/BNFabb88.xml

```diff
@@ -0,0 +1,381 @@
+                                    <width unit="mm" cert="medium">200</width>
+                                </extent>
+                                <foliation>Foliation in the upper right corner of the
+                                    folia.</foliation>
+                                <collation>
+                                    <note>Due to the nature of modern library rebinding, the quire
+                                        quire structure has become invisible or lost.</note>
+                                </collation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="18">
+                                    <ab type="pricking">Pricking is partly visible.</ab>
+                                    <ab type="ruling">Ruling is not visible.</ab>
+                                    <ab type="ruling" subtype="pattern" cert="low"
+                                        >1A-1A-1A1A/0-0/0-0/C</ab>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">18th-century script</seg>
+                                <seg type="ink">Black, red.</seg>
+                                <desc>Careful, irregular; by a good scribe. <persName
+                                        ref="PRS1071dAbbadi"/> describes the handwriting as <foreign
+                                        xml:lang="fr">ʻguịlḥ moderneʼ</foreign>. </desc>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <locus target="#2v"/>
+                            </decoNote>
+                            <decoNote xml:id="d2" type="miniature">
+                                <locus target="#3v"/>
+                                <desc><ref type="authFile" corresp="AT1042HagiographicScene"
+                                            ><persName ref="PRS9848Walatta"/>Walatta Ṗeṭros taming
+                                        the hippopotamuses on Lake Tana.</ref></desc>
+                                <note>Interpretation of the miniature is taken from <bibl>
+                                        <ptr target="bm:BoscWion2010Abbadie"/>
+                                        <citedRange unit="page">90</citedRange>
+                                    </bibl>, containing a colorful reproduction of the miniature.
+                                </note>
+                            </decoNote>
+                            <decoNote xml:id="d3" type="miniature">
+                                <locus target="#4r"/>
```

Comment:
the description for d3 disappeared completely

Reply:
> the description for d3 disappeared completely

yes, this is how I understood your comment saying "If you are not sure what you are seeing you do not have to assign an art theme or produce any other mark up". Did I misunderstand it and should have left something of the description?